presto-hive/src/main/java/com/facebook/presto/hive/parquet/RichColumnDescriptor.java

```diff
@@ -43,4 +46,14 @@ public boolean isRequired()
     {
         return required;
     }
+
+    public void setHiveType(HiveType hiveType)
```

Comment:
It's possible to pass this in the constructor as far as I can see in the code. So, please do so, mark `hiveType` as final and mark `RichColumnDescriptor` as `Immutable`. Also please update the comment in this class.